data/example-data-mongodb-book/src/main/java/io/americanexpress/data/book/DataBookApplication.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.americanexpress.data.book;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+
+/**
+ * DataBookApplication is the Spring Boot main class to start the data application.
+ */
+@SpringBootApplication
+@EnableConfigurationProperties
+public class DataBookApplication {
```

Comment:
Maybe we can name this MongoDataBookApplication, but not sure if we just want to keep the name consistent.

Reply:
I think we should keep the name consistent.